src/js/lineChartTimeSeriesArea.js

```diff
@@ -132,7 +140,22 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                 "transform": "translate(0)"
             });
         });
+    };
 
+    // Uses D3's default transition; this results in the
+    // "wriggle" described at https://bost.ocks.org/mike/path/
+    floe.chartAuthoring.lineChart.timeSeries.area.updateArea.defaultTransition = function (chartLineAreaPaths, area, width, transitionLength) {
```

Comment:
Do you think if it makes sense to state in the code comment that this is for the "real time" use case, such as working with the dynamic nexus data input.